dashboard/app/models/pd/application/teacher_application_base.rb

```diff
@@ -65,12 +64,6 @@ class TeacherApplicationBase < ApplicationBase
       pd_workshop_id
     )
 
-    def scholarship_status_valid
-      unless scholarship_status.nil? || Pd::ScholarshipInfoConstants::SCHOLARSHIP_STATUSES.include?(scholarship_status)
```

Comment:
did you check if these constants are used incorrectly anywhere else?

Reply:
good thought -- I just went through and double checked that all uses of `Pd::ScholarshipInfoConstants::something` are of individual constants (not lists) that still exist (eg, `Pd::ScholarshipInfoConstants::NO`). 

I think I may have confused myself the first time around when I used things like `Pd::ScholarshipInfo::YES_CDO`, which is possible because `Pd::ScholarshipInfoConstants` is included at the top of the `Pd::ScholarshipInfo` model. I've moved references to the more confusing use of constants to the less confusing version in my most recent commit.